dbt/adapters/athena/impl.py

```diff
@@ -237,17 +236,17 @@ def clean_up_partitions(self, relation: AthenaRelation, where_condition: str):
             self.delete_from_s3(partition["StorageDescriptor"]["Location"])
 
     @available
-    def clean_up_table(self, relation: AthenaRelation):
+    def clean_up_table(self, relation: AthenaRelation) -> None:
         table_location = self.get_glue_table_location(relation)
 
-        # this check avoid issues for when the table location is an empty string
-        # or when the table do not exist and table location is None
+        # this check avoids issues for when the table location is an empty string
+        # or when the table does not exist and table location is None
         if table_location:
             self.delete_from_s3(table_location)
 
     @available
     def quote_seed_column(self, column: str, quote_config: Optional[bool]) -> str:
-        return super().quote_seed_column(column, False)
```

Comment:
Since we call super there should be no need to cast as str, except if the super method is poorly typed ?

Reply:
yeah, the problem is in method from `base` module
```
dbt/adapters/athena/impl.py:249: error: Returning Any from function declared to return "str"  [no-any-return]
```